### PR TITLE
MODINV-119 Create endpoint to save collection of instances

### PR DESCRIPTION
**Pull request by Igor-Gorchakov**
Copy of https://github.com/folio-org/mod-inventory/pull/115

### CODE REVIEW

**Comment by marcjohnson-kint** [comment]
Why was a fresh pull request raised?

**Comment by marcjohnson-kint** [comment]
> Because of last pull request has a lot of formatting changes included. I prepared a fresh one to make changes easier to review.

Ok, thank you

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -2,6 +2,23 @@
...
   "name": "Inventory Module",
   "provides": [
+    {
+      "id": "inventory-batch",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": ["POST"],
+          "pathPattern": "/inventory/instances/batch",
+          "permissionsRequired": ["inventory.instances.item.post.batch"],
```
Comment (marcjohnson-kint) [inline descriptors/ModuleDescriptor-template.json]: We've used different conventions for specialist endpoints like this. Please only take this as a alternative suggestion, if you prefer what you already have please just resolve without changing

```suggestion
          "permissionsRequired": ["inventory.instances.batch.post"],
```

**Comment by marcjohnson-kint** [comment]
> > Because of last pull request has a lot of formatting changes included. I prepared a fresh one to make changes easier to review.
> 
> Ok, thank you

This pull request seems to also include the same re-formatting. Have I missed something?

File: src/test/java/api/InstancesApiExamples.java [hunk 2]
```diff
@@ -187,6 +189,107 @@ public void canCreateAnInstanceWithAnID()
     selfLinkShouldBeReachable(createdInstance);
   }
 
+  @Test
```
Comment (marcjohnson-kint) [inline src/test/java/api/InstancesApiExamples.java]: I think these tests could benefit from some whitespace. Maybe separating out the setup of each record, the batch submission and the checks on the response into groups.

Comment (Igor-Gorchakov) [reply src/test/java/api/InstancesApiExamples.java]: Done

**Comment by Igor-Gorchakov** [comment]
> I haven't reviewed the implementation much, however I'm keen not to hold up this work any further.

Thanks for understanding our context

**Pull request merged**